src/core/icons.js

```diff
@@ -1,5 +1,5 @@
 import { loadSprites } from "./scripts";
-import sprites from "./sprites.svg";
+import sprites from "../../core/sprites.svg";
```

Comment:
`sprites.svg` is now built before Storybook launches, so we know this will be here. The `sprites.svg` in `src/core` shouldn't be here (since this is a dynamically generated asset), so has been removed. This way, we know that Storybook always has a spritemap for the latest icons.